src/pages/Chat/index.jsx

```diff
@@ -54,15 +61,25 @@ const Chat = () => {
     }
   }
 
+  const handleKeyDown = event => {
+    if (event.key === 'Enter') {
+      handleSubmit()
+    }
+  }
+
   return (
     <Layout
       SendForm={
-        <FooterChatInput cols={'0 0 100%'}>
+        <FooterChatInput 
+// @ts-ignore
+        cols={'0 0 100%'}>
           <Input
+            // @ts-ignore
             bgColor="gray"
             placeholder="Escribe aqui..."
             value={message}
             onChange={e => setMessage(e.target.value)}
+            onKeyDown={handleKeyDown}
           />
           <img src={sendChat} alt="send Chat" onClick={() => handleSubmit()} />
```

Comment:
Why not use a <form></form> with a button submit instead of use onKeyDown and onClick?

```HTML
<form onSubmit="handleSubmit()">
   <input ...>
   <button type="submit">
</form>
```

This will work if the user presses the button or if he presses Enter